### PR TITLE
 Milestone 1 : Blog app | Create Data Model #1 

### PR DESCRIPTION
**Pull request by betses**
### In this milestone I have:

-    Install Ruby on Rails framework.
-    Use Ruby Gems as a software package system.
-    Understand what ORM is.
-    Use database migration files to maintain database schema.
-    Install and check linters.
-    Add a descriptive README file with the new version 2.0.

### CODE REVIEW

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @betses , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has failing linters, kindly fix it

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @betses,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ You used Gitflow
✅ No linter error

## Required Changes ♻️
_Check the comments under the review._

- [ ] For the create_posts migration, you did not set up a reference key, as a post is created by a user, therefore, the id of the author who created the post has been referenced as a foreign key in the migration. You could create a separate migration for that 👍 
- [ ] Also on the create_comments migration, a post foreign key has to be referenced for which a comment is created on 👍 
- [ ] Kindly add documentation to your project on the README file. It's having the default README generated by rails when you created the app. Kindly use the Microverse-recommended professional README template for this project. 👍 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md [hunk 1]
```diff
@@ -1,285 +1,24 @@
-<a name="readme-top"></a>
```
Comment (SajjadAhmad14) [inline README.md:1]: Please add a professional README as it's also requested by the previous code reviewer. You may use [this](https://github.com/microverseinc/readme-template) tempalate from Microverse.

File: db/migrate/20221124073632_create_users.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class CreateUser < ActiveRecord::Migration[7.0]
```
Comment (SajjadAhmad14) [inline db/migrate/20221124073632_create_users.rb]: There is a migration error so I was not able to migrate the database. Kindly make sure that there is no database errors.

mig

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Betsegaw,
### Great work on making the changes requested by a previous reviewer 👏 
### You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.
## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Reem-lab — APPROVED** [review]
# Hi @betses, 👋🏻👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🚀🔥🎉🎉 


## Highlights
- No Linters error ✅
- Followed Git flow ✅
- Descriptive PR 🚀✅
- Great working on the functionality 🔥🚀🎉

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**